addon/locales/en/components/map-commands-dialogs/go-to.js

```diff
@@ -2,7 +2,8 @@ export default {
   'caption': 'Go to specified point',
   'error-message': {
     'caption': 'Go to error',
-    'content': 'Specified coordinates are wrong & can\'t be successfully converted into numbers'
+    'content': 'Specified coordinates are wrong & can\'t be successfully converted into numbers',
+    'large-coords': 'Specified coordinates doesn\'t exist in allowable map area'
```

Comment:
For some reason real captions in english version of go-to dialog's UI doesnt't correspond to locales in this file: 
![image](https://cloud.githubusercontent.com/assets/7602229/26236278/f4c22732-3c88-11e7-8925-8370c47f0c5d.png)